Add paragraph about creating app on message creation (#111)
svix/svix-docs: docs/overview.mdx

It's recommended to create an application (using the API) for each of their customers when they sign up to your service. You can however also create it "lazily", and only do it when they enable webhooks.

> You also have the option of creating an application the first time you send a message to it. When sending a message you can, optionally, send an application specification along with it. If the application does not exist yet it will be created; if it does then the application specification is ignored.

Reviewer: ```suggestion
Alternatively, you can create an application the first time you send a message to it. When sending a message you can, optionally, send an `application` property along with it. If the application does not exist yet it will be created; if it does then the `application` property will be ignored.
```
